plugins/drivers/raw-exec-plugin/Makefile

```diff
@@ -0,0 +1,11 @@
+generate:
```

Comment:
any reason not to call it just raw-exec rather than raw-exec-plugin since its already under plugins? 

This can be changed later too, so not a blocker to merge

Reply:
We have `/plugins/drivers/raw-exec-plugin/plugin-go-grpc/raw_exec/` - this layout is directly pulled from https://github.com/hashicorp/go-plugin/tree/master/examples/grpc

I'll merge this for now and will open an issue to restructure this layout to something that is less verbose/deeply nested.